src/services/db/GitHubService.ts

```diff
@@ -132,13 +133,13 @@ export default class GitHubService {
       fileName,
       directoryName,
       isMedia = false,
-      branchName = STAGING_BRANCH,
+      branchName,
```

Comment:
this is abit weird - why is the default arg removed? we should just add this back to maintain behaviour. 

my concern here is that some caller doesn't pass an arg and relies on the default, which is no longer true here

Reply:
pls do this before merge - i scared of regression + it's a fast change

Reply:
sure, ill add this in, but just to close the loop here this is not needed as 
1. only repoService calls githubService directly
2. repoService adds in this required param (both are in ts so is easier to guarentee)

Reply:
ad0ee4c468d86c247401e2fc7fd49b350ad5d72f